### ISSUE TITLE
Should this package use the latest version of the app-android-sdk ?

### ISSUE THREAD

**Issue by juansimon28** [Appboy/appboy-react-sdk#123]
I was integrating this package into our RN project and I faced an issue while adding this service:

```
<service android:name="com.braze.push.BrazeFirebaseMessagingService"
  android:exported="false">
  <intent-filter>
    <action android:name="com.google.firebase.MESSAGING_EVENT" />
  </intent-filter>
</service>
```

As the documentation required to handle push notifications and open intents
[step-1-register-braze-firebase-messaging-service](https://www.braze.com/docs/developer_guide/platform_integration_guides/android/push_notifications/integration/standard_integration/#step-1-register-braze-firebase-messaging-service)

## The problem is

This change `com.braze.push.BrazeFirebaseMessagingService` was introduced in the 15.0.0 version [Android SDK 15.0.0 Release notes](https://www.braze.com/docs/developer_guide/platform_integration_guides/android/changelog/#1500), along with a bunch of other renames.

Right now, the android-sdk version is 14.0.1, and to properly use the service we need to use this:
`com.appboy.AppboyFirebaseMessagingService` instead of `com.braze.push.BrazeFirebaseMessagingService`

**Comment by radixdev** [comment]
Hi @juansimon28 ,

We will be updating to 15.0.0 and will update this issue after that is done.

**Comment by juansimon28** [comment]
Cool @radixdev thanks!

**Comment by wesleyorbin** [comment]
Hi @juansimon28! We recently released v1.32.0, which updated the Android SDK to 15.0.0. I'm closing out this issue, but feel free to reopen if your issue is not solved. Thanks!
